src/docfx/docset/Docset.cs

```diff
@@ -27,6 +27,11 @@ internal class Docset
         /// </summary>
         public Repository Repository { get; }
 
+        /// <summary>
+        /// Gets the docset name
+        /// </summary>
+        public string Name => Config.Name;
```

Comment:
why not just use `docset.config.name`. people may forward every other property to `Docset`. I intend `Docset` to only contain properties that need further processing.